samples/unity-of-bugs/Assets/Resources/Sentry/SentryOptions.asset

```diff
@@ -47,7 +47,7 @@ MonoBehaviour:
   <MacosNativeSupportEnabled>k__BackingField: 1
   <LinuxNativeSupportEnabled>k__BackingField: 1
   <Il2CppLineNumberSupportEnabled>k__BackingField: 1
-  <OptionsConfiguration>k__BackingField: {fileID: 11400000, guid: 407b4d5f2fef2c845bf2a3dcdf6b00fb,
+  <RuntimeOptionsConfiguration>k__BackingField: {fileID: 11400000, guid: 407b4d5f2fef2c845bf2a3dcdf6b00fb,
```

Comment:
I'm not entirely happy about the rename of the option variable because that means when users upgrade, they'll silently lose their existing configurations (they won't be run anymore), doesn't it?

Reply:
Reverted the renaming.